projects/cdk/observables/control-value.ts

```diff
@@ -13,6 +13,10 @@ export function tuiControlValue<T>(
             throw new TuiValueChangesException();
         }
 
-        control.valueChanges.pipe(startWith(control.value)).subscribe(subscriber);
+        const subscription = control.valueChanges
+            .pipe(startWith(control.value))
+            .subscribe(subscriber);
+
+        return () => subscription.unsubscribe();
```

Comment:
You can just add `return`, the subscription is a valid teardown logic.

Reply:
fixed